nano/node/bootstrap/bootstrap_frontier.cpp

```diff
@@ -320,11 +320,11 @@ void nano::frontier_req_server::next ()
 		auto transaction (connection->node->store.tx_begin_read ());
 		for (auto i (connection->node->store.latest_begin (transaction, current.number () + 1)), n (connection->node->store.latest_end ()); i != n && accounts.size () != max_size; ++i)
 		{
-			nano::account_info const & info (i->second);
-			if (!skip_old || (now - info.modified) <= request->age)
+			auto state (connection->node->ledger.account_state (transaction, i->second));
```

Comment:
This request seems like extra i/o if `request->age == std::numeric_limits<decltype (request.age)>::max ()` (default). Then `i->second.head` can be used

Reply:
Actually it can be like
```
bool send_frontier (true);
if (skip_old)
{
	auto state (connection->node->ledger.account_state (transaction, i->second));
	send_frontier = (now - state.modified ()) <= request->age;
}
if (send_frontier)
{
	nano::account const & account (i->first);
	nano::account_info const & info (i->second);
	accounts.emplace_back (account, info.head);
}
```